build-linux/deb/postinst

```diff
@@ -30,11 +30,27 @@
 
 set -e
 
+install_init () {
+    cp /var/lib/pdagent/scripts/pdagent.init /etc/init.d/pdagent
+    chmod +x /etc/init.d/pdagent
+    update-rc.d -f pdagent defaults
+    service pdagent start
+}
+
+install_systemd () {
+    touch /var/log/pdagent/pdagentd.log
+    chmod 640 /var/log/pdagent/pdagentd.log
+    chown -R pdagent:pdagent /var/log/pdagent/pdagentd.log
```

Comment:
In 16.04 pdagentd.log was being created with mode 644, so we have change it here. There doesn't seem to be a way to set the mode through the logging module in python.